out.py

```diff
@@ -15,18 +15,19 @@ class _CursorInfo(ctypes.Structure):
 
 
 def error(message: str, context: Any = __UNIT) -> None:
-    print(message, file=sys.stderr)
+    println(danger(message), file=sys.stderr)
     if context is not __UNIT:
         print(context, file=sys.stderr)
 
 
 RESET = "\033[0m"
-PRIMARY = "\033[95m"
+PRIMARY = "\033[92m"
 SECONDARY = "\033[94m"
+DANGER = "\033[91m"
 
 
-def println(*args, end: str = "\n") -> None:
-    sys.stdout.write(" ".join(map(str, args)) + end)
+def println(*args, end: str = "\n", file: Any = sys.stdout) -> None:
+    print(" ".join(map(str, args)), end=end, file=file)
```

Comment:
If you use the built int print function, `...join(map..` is not required. You can just pass the args directly